src/components/card/card.scss

```diff
@@ -0,0 +1,211 @@
+
```

Comment:
- [x] RTL tweaks taken care of
- [x] Emailed stakeholders in sizing customization, will update into variables based on their feedback